web/docs/tql2/functions/all.md

```diff
@@ -20,11 +20,10 @@ A list of boolean values.
 ### Check if all values are true
 
 ```tql
-from [
+from \
```

Comment:
Do you want to change this to not use `\\`? I thinks it doesn't look great when looking at the rendered documentation. Here and in many other places, you can move everything to a single line now. 

For examples where this doesn't look good, maybe put the first event into the first line? Don't really have a strong opinion on this though.